packages/clients/textLocator/src/plugins/GeometrySearch/components/GeometrySearch.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <v-card class="text-locator-search-card">
+    <DrawMode />
+    <v-card-title class="text-locator-collapse">
+      {{ $t('common:plugins.geometrySearch.results.title') }}
+    </v-card-title>
+    <v-card-text>
+      <ViewToggle />
+      <v-treeview
+        v-if="treeViewItems.length"
+        dense
+        hoverable
+        color="info"
+        :items="treeViewItems"
+      >
+        <template #label="{ item }">
+          <v-badge color="info" inline left class="text-locator-result-badge">
+            <template #badge>
+              <v-tooltip left>
+                <template #activator="{ on }">
+                  <span v-on="on">{{ item.count }}</span>
+                </template>
+                <span>{{ $t(tooltipKey(item)) }}</span>
+              </v-tooltip>
+            </template>
+            {{ item.name }}
+            <template v-if="item.type === 'toponym' && item.feature">
+              &nbsp;
+              <ResultInfo :feature="item.feature" direction="left"></ResultInfo>
+            </template>
+          </v-badge>
+        </template>
+        <template #append="{ item }">
```

Comment:
|Icon\Type|Toponym|Literature|
|-|-|-|
|Map|Zoom to location|Zoom to all (currently loaded)* referenced geometries**|
|Magnifying Glass|Find all intersecting geometries|Find all referenced geometries (NOT IMPLEMENTED)|

\* There will be other paths to loading later; currently, only geometries selected via map and search are available. Later, all geometries to a piece of literature will be fetchable.

\*\* Only available when sorting by literature.

Maybe this table helps this discussion. I tried different icons for the two columns, but it didn't really help with what the methods do, since both rows do a samey thing that differs in its functionality by what kind of entity the line belongs to.

Do you have any suggestion on what icons may fit *and* help in understanding what happens? When trying other icons, I only managed to make the UI look somewhat confusing/overloaded. I guess that, for the user to understand what those buttons do, the tooltip has to be read.